docs/release-notes/cloud/cloud-2023-02.md

```diff
@@ -26,7 +26,6 @@ These release notes are for the Codacy Cloud updates during February 2023.
 -   Fixed an issue that prevented organization owners from receiving the email notification with new requests to join the Codacy organization when the requester user didn't have a name set up. (PLUTO-365)
 -   Fixed an unexpected error when editing a coding standard and moving back to the **Select languages** step and then forward to the tool selection step. (IO-434)
 -   Fixed an issue that caused Codacy to send incorrect analysis status notifications to Git providers when failing to reanalyze a pull request. (IO-424)
--   Improved the reporting of analysis and coverage status to Git providers when new commits are pushed during an ongoing analysis. (IO-341)
```

Comment:
This release note was included here by mistake since it had already been announced in the [January release notes](https://github.com/codacy/docs/blob/master/docs/release-notes/cloud/cloud-2023-01.md?plain=1#L38).